diff --git a/node_modules/react-native-text-input-mask/android/build.gradle b/node_modules/react-native-text-input-mask/android/build.gradle
index f25dcb0..4a2952e 100644
--- a/node_modules/react-native-text-input-mask/android/build.gradle
+++ b/node_modules/react-native-text-input-mask/android/build.gradle
@@ -50,6 +50,6 @@ repositories {
 
 dependencies {
     implementation 'com.facebook.react:react-native:+'
-    implementation 'com.redmadrobot:input-mask-android:7.2.6'
+    implementation 'com.redmadrobot:input-mask-android:6.1.0'
     implementation "org.jetbrains.kotlin:kotlin-stdlib:${project.ext.kotlinVersion}"
 }
